newIDE/app/src/UI/Search/ListSearchResults.js

```diff
@@ -122,7 +122,13 @@ export const ListSearchResults = <SearchItem>({
 
             return (
               <Grid
-                ref={grid}
+                ref={el => {
+                  if (el) {
```

Comment:
Interesting! I think calling `recomputeGridSize` is the right thing to do (per the doc: "This function should be called if dynamic column or row sizes have changed but nothing else has" => exactly our situation).

Can you check if:
- this works well on everything using a ListSearchResults (i.e: game examples too and the game showcase).
- if instead of "hijacking" the ref callback function, we could use a `React.useLayoutEffect`? With empty dependencies to simulate a `componentDidMount`?

According to the docs, "React will call the ref callback with the DOM element when the component mounts, and call it with null when it unmounts.". 
So we should be able to replace it with a useLayoutEffect as it "fires in the same phase as componentDidMount". 

Let me know if it works or not

Reply:
- checked on all ListSearchResults, it looks great :+1:
- I tried `useLayoutEffect`, unfortunately it gets called too early, the grid reference hasn't been set yet, it is still at its initial value `null`

Reply:
Thanks for checking!